src/Components/MoodRating/MoodRating.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState, useContext } from 'react'
+import style from './MoodRating.module.scss'
+import { ViewContext } from '../../Context/ViewContext'
+import { SessionContext } from '../../Context/SessionContext'
+import { postSession } from '../../apiCalls'
+import moodIcon1 from '../../assets/moodIcons/moodIcon1.png'
```

Comment:
Nice work on these mood icons @Abdeboskey !!! I really like how they fit into the theme of the rest of the app.